tutorials/cp-launchpad-federation-consumption/cp-launchpad-federation-consumption.md

```diff
@@ -196,10 +196,11 @@ Now you are done in SAP BTP cockpit.
 
     ![App Manage outgoing checks](29-manage-checks.png)
 
+<!---
```

Comment:
If you don't need a passage in your tutorial, please remove it. Commenting out parts of a tutorial will make the code messy in the long run.

Reply:
This part only needed to be removed for some time, as we are currently facing an issue with it. Hope this will be fixed and we can then remove the comment.